app/src/test/java/com/x8bit/bitwarden/ui/vault/feature/item/util/CipherViewExtensionsTest.kt

```diff
@@ -264,6 +273,7 @@ class CipherViewExtensionsTest {
     fun `toViewState should transform empty Secure Note CipherView into ViewState Secure Note Content`() {
         val cipherView = createCipherView(type = CipherType.SECURE_NOTE, isEmpty = true)
         val viewState = cipherView.toViewState(
+            previousState = null,
```

Comment:
I see a lot of tests where this is `null`. Can we add some where it is _not_ `null`?

Reply:
Added!